### DISCUSSION TITLE
Upgrade to deepin-daemon and deepin-api error

### DISCUSSION THREAD

**Discussion by Cardellino** [linuxdeepin/developer-center#3330] [Q&A | 问答板块] [answered]
Hello, yesterday (07/25/22) I upgraded my Arch Linux system.
After that I checked that the bar didn't work anymore.
Just no reaction. All icons are simply not accessible.
The upgrade:
deepin-daemon-5.14.44-2-x86_64.pkg.tar.zst
deepin-api-5.5.25-1-x86_64.pkg.tar.zst

After I downgrade to:
deepin-daemon-5.13.49-1-x86_64.pkg.tar.zst
deepin-api-5.4.11-1-x86_64.pkg.tar.zst
everything went again..

My system:
ArchLinux x86_64
5.18.14-arch1-1
Deepin 20.2.2

Should any of you have the same problem
Many greetings

**Comment by BLumia** [comment]
I got the same issue, I thought it was something wrong on my side. I'll convert this as an issue for better issue tracking, thanks for the report!

**Reply by BLumia** [reply]
btw, are you also using `deepin-wm` instead of `deepin-kwin`? (For the record, I'm using `deepin-wm`)

**Reply by Cardellino** [reply]
Using deepin-wm-1.9.38-6

**Comment by zccrs** [comment]
(dde-session-daemon:235138): GLib-GIO-ERROR **: 15:21:33.988: Settings schema 'com.deepin.dde.power' does not contain a key named 'high-performance-enabled'
SIGTRAP: trace trap
PC=0x7f13202f5068 m=19 sigcode=128
signal arrived during cgo execution

**Comment by BLumia** [comment, marked as answer]
`deepin-desktop-schemas` has been updated to `5.10.6-2` by @felixonmars which will hopefully resolve this issue. I couldn't test it right now for some reason and will test on my side later. Could you try if updating `deepin-desktop-schemas`  to `5.10.6-2` helps? Thanks!

**Reply by Cardellino** [reply]
Hello Blumia, started an upgrade a few minutes ago.
Looks good with no problems.
2 x system started..
It's all ok...

Thank you for the quick support
Have a nice day

**Reply by Cardellino** [reply]
Another thing, the programs that were started are not visible in the bar!
For example Google Earth -> Icon
See photo!
[Bildschirmaufnahme_Auswahlbereich_20220726141828](https://user-images.githubusercontent.com/73848158/181004672-4b79bb84-6f57-4599-acd4-e0bf14b86eee.png)

**Reply by BLumia** [reply]
Please create seprated new issue report for different issue so it can be easier for issue tracking. Issues can be reported [here](https://github.com/linuxdeepin/developer-center/issues). Thanks!
